dm/pkg/schema/tracker.go

```diff
@@ -366,6 +372,12 @@ func (tr *Tracker) Reset() error {
 
 // Close close a tracker.
 func (tr *Tracker) Close() error {
+	if tr == nil {
+		return nil
+	}
+	if !tr.closed.CAS(false, true) {
+		return nil
+	}
 	tr.se.Close()
 	tr.dom.Close()
 	if err := tr.store.Close(); err != nil {
```

Comment:
Do we need to remove `tr.storePath` before we start? I'm afraid an ungraceful stop may not remove the storage path.

Reply:
TempDir will return a newly created folder by appending suffix

Reply:
Oh user may need to manually clean it to release disk space. Hope it's not occupy large space.